src/style.css

```diff
@@ -0,0 +1,85 @@
+body {
+  font-family: "Montserrat", sans-serif;
+  overflow: hidden;
+  margin: -21px 0 0 0;
+}
+
+.container {
```

Comment:
This class/selector is too generic. This is a big problem in CSS. You might have some problems in the future with these generic classes. The biggest problem is to get duplicated classes, so you'll get some styles applied to a component that you don't want.

There some options to deal with it. The easiest one (based on learning and refactoring) is to use BEM methodology, there is a good article about it: https://medium.com/trainingcenter/bem-em-5min-f5c80fd23439